### DISCUSSION TITLE
Disable Sidebar

### DISCUSSION THREAD

**Discussion by tejngx** [aclap-dev/video-downloadhelper#1334] [Help and Q&A]
### Your question

Sidebar always opened automatically when launching Firefox. How to disable that.

### Addon & CoApp version and details

version: 9.1.0.44
channel: stable
build date: 2024-09-04
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-GB
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:131.0) Gecko/20100101 Firefox/131.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "M:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Admin",
  "converterBinary": "M:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Search this forum for other discussions of the sidebar.  This is something other people have encountered.  You should have searched before you posted.
